src/components/commentArticle/LikeUnlikeReplies/index.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { ToastContainer, toast } from 'react-toastify';
+import { Popup } from 'semantic-ui-react';
+import style from './LikeDislikeReplies.scss';
+import likeAction from '../../../redux/actions/commentArticle/likeComment';
+import fetchLikes from '../../../redux/actions/commentArticle/likeComment/fetchAllLikes';
+
+export class LikeDislikeRepliesComponent extends Component {
+state = {
+  likesCount: 0,
+  peopleWhoLiked: '',
+  actionMethod: 'POST',
+};
+
+componentDidMount() {
+  const { fetchLikes, commentId } = this.props;
+  fetchLikes(commentId);
+}
+
+  UNSAFE_componentWillReceiveProps = (nextProps) => {
+    const { counts, commentId } = nextProps;
+    (counts || []).forEach((count) => {
+      if (Number(count.commentId) === Number(commentId)) {
+        this.setState((prevState) => ({ ...prevState, likesCount: count.likesCount }));
+        this.setState((prevState) => ({ ...prevState, peopleWhoLiked: count.formattedLikeInfo }));
+      }
+    });
+  };
+
+  handleClick = (isLikedByMe, likesCount) => {
+    const { history, location } = this.props;
+    if (!localStorage.getItem('token')) {
+      history.push(
+        {
+          pathname: '/login',
+          state: { from: location },
+        },
+      );
+      return;
+    }
+    const {
+      commentId, likeAction,
+    } = this.props;
+    const method = isLikedByMe ? 'PUT' : 'POST';
+    likeAction(commentId, method)
+      .then((res) => {
+        if (res.payload.status === 'success') {
+          this.setState((prevState) => ({
+            ...prevState,
+            likesCount: isLikedByMe
+              ? likesCount - 1
+              : likesCount + 1,
+            peopleWhoLiked: isLikedByMe
+              ? prevState.peopleWhoLiked.toLowerCase().replace('you', '').replace(',', '')
+              : `You, ${prevState.peopleWhoLiked.toLowerCase().replace('you', '')}`,
+          }));
+        }
```

Comment:
This logic must be in the actions. You then should add the people who liked in the global state. That's why when we were fetching the comments again the people who liked were not being updated.